### ISSUE TITLE
Ensure use of <ul> and <ol> conforms to WCAG usage guidelines

### ISSUE THREAD

**Issue by sidramatik** [CityofToronto/wp-open-data-toronto#7]
i.e. the immediate element following `<ul>` or `<ol>` should always be `<li>`. 

See sidebar 
![image](https://user-images.githubusercontent.com/498765/46163647-bfc47000-c259-11e8-8b41-9abf2e3c79b6.png)

i.e.
Instead of 
`<ul><div class="hello"><a href="#">hello world</a></div></ul>`
should be
`<ul><li class="hello"><a href="#">hello world</a></li></ul>`
